docs/components/Navigation.tsx

```diff
@@ -22,6 +20,19 @@ function Navigation(props) {
     });
   }
 
+  const lastItem = props.items[props.items.length - 1];
+  if (lastItem?.id !== "contextual-enterprise") {
```

Comment:
Where is this contextual-enterprise string coming from?

Reply:
It's an ID that exists here only so we only add this once. Otherwise, every render would add another Enterprise link to the list! This ensures there's just one (with this ID)